### PR TITLE
add construction hash

### PR DESCRIPTION
**Pull request by jgimeno**

### CODE REVIEW

**Review by ilgooz — APPROVED** [review]
:confetti_ball:

File: cosmos/launchpad/construction_hash_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"github.com/cosmos/cosmos-sdk/simapp"
+
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLaunchpad_ConstructionHash(t *testing.T) {
+	expectedHash := "6f22ea7620ebcb5078d244f06e88dd26906ba1685135bfc34f83fefdd653198a"
+
+	open, err := os.Open("./testdata/signed-tx.json")
```
Comment (sahith-narahari) [inline cosmos/launchpad/construction_hash_test.go:22]: any reason for not using `ioutil.ReadFile` directly

**Review by sahith-narahari — APPROVED** [review]
lgtm, left a minor suggestion

File: cosmos/launchpad/construction_hash_test.go (hunk 1 above)
Comment (jgimeno) [reply cosmos/launchpad/construction_hash_test.go]: no, thanks! Updated!

**Pull request merged**